velox/common/file/CMakeLists.txt

```diff
@@ -21,3 +21,11 @@ target_link_libraries(file velox_exception ${GTEST_BOTH_LIBRARIES} ${GLOG}
 add_executable(file_test FileTest.cpp)
 add_test(file_test file_test)
 target_link_libraries(file_test file)
+if(VELOX_ENABLE_S3)
```

Comment:
would it make sense to move the s3 stuff to `velox/common/file/s3/` or even something like `velox/adapters` and here just
```
if(VELOX_ENABLE_S3)
   add_subdirectory(s3)
endif()
```